src/OrchardCore.Modules/OrchardCore.Google/GoogleAuthenticationAdminMenu.cs

```diff
@@ -45,25 +64,36 @@ public GoogleAnalyticsAdminMenu(IStringLocalizer<GoogleAnalyticsAdminMenu> local
 
         public Task BuildNavigationAsync(string name, NavigationBuilder builder)
         {
-            if (string.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
+            if (!NavigationHelper.IsAdminMenu(name))
             {
-                builder.Add(S["Configuration"], configuration => configuration
-                        .Add(S["Settings"], settings => settings
-                            .Add(S["Google Analytics"], S["Google Analytics"].PrefixPosition(), settings => settings
-                            .AddClass("googleAnalytics").Id("googleAnalytics")
-                            .Action("Index", "Admin", new { area = "OrchardCore.Settings", groupId = GoogleConstants.Features.GoogleAnalytics })
-                                .Permission(Permissions.ManageGoogleAnalytics)
-                                .LocalNav())
-                            )
-                        );
+                return Task.CompletedTask;
```

Comment:
Since there's a single statement after this, an early return doesn't look like it's any nicer with a duplicated `return Task.CompletedTask;`. Same for the others in this class.

Reply:
Its just a taste. doing the return early on, saves us 1 indentation. I think it is much cleaner when you have less indentation.

Reply:
I agree if there's a lot of code afterward. However, here it's one statement.

Reply:
For me I think it is best to keep consistence across the board.

Reply:
OK, let's leave it like that.